project1/css/index.css

```diff
@@ -26,13 +27,81 @@ time, mark, audio, video {
 /* HTML5 display-role reset for older browsers */
 article, aside, details, figcaption, figure, 
 footer, header, hgroup, menu, nav, section {
-	display: block;
+	
 }
 body {
 	line-height: 1;
 }
+
+
+header nav  {
+margin-top: 40px;
+
+}
+li {
+	display: inline-block;
+	padding-right:60px;
+	padding-top:20px;
+}
```

Comment:
The links should of had a color of gray, just keeping inline with the image given.